app/controllers/audio_segments_controller.rb

```diff
@@ -40,6 +40,16 @@ def destroy
     @audio_segment.destroy
   end
 
+  # GET /audio_segments/search
+  def search
+    if(params.has_key?(:met_start) and params.has_key?(:met_end))
+      render json: AudioSegment.where("met_end > ? AND met_start < ?", params[:met_start], params[:met_end])
```

Comment:
Can you confirm that this statement is protected against SQL injection? Met start needs to be asserted as an int for instance